### PR TITLE
Document directory decommission and meta2-mover

### PR DESCRIPTION
**Pull request by acornard**

### CODE REVIEW

File: doc/source/admin-guide/operations_move_meta2.rst [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+The base name is composed by CID and SEQ. The CID is before "." and SEQ is after.
+
+If the base name is 43839DACDD060FA939FAE2714A60640BDC5AFFDDBE8C494BDAB7AA994C3190A5.1,
+
+the CID is 43839DACDD060FA939FAE2714A60640BDC5AFFDDBE8C494BDAB7AA994C3190A5 and the SEQ is 1
+
+You can launch mover by running
+  .. code-block:: console
+
+    # oio-meta2-mover <namespace> <IP> <CID>
```
Comment (murlock) [inline doc/source/admin-guide/operations_move_meta2.rst]: It is `# oio-meta2-mover <namespace> <CID> <IP source>` ?

File: doc/source/admin-guide/operations_decommission_meta1.rst [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+Find information about the service you want to decommission.
+By running ``openio cluster list meta1`` you will get the list of all meta1 service address accompanied by their volume path.
+
+
+
+Launch decommissioning
+~~~~~~~~~~~~~~~~~~~~~~
+
+Run ``openio directory decommission <IP>``, the IP is the the address of the meta1.
```
Comment (jkasarherou) [inline doc/source/admin-guide/operations_decommission_meta1.rst]: `IP` is not correct: use `address` or `IP:PORT`.

Comment (jkasarherou) [inline doc/source/admin-guide/operations_decommission_meta1.rst]: Show at least one output example of decommission

File: doc/source/admin-guide/operations_move_meta2.rst [hunk 3]
```diff
@@ -0,0 +1,36 @@
+====================
+Move a meta2 service
+====================
+
+Preparation
+~~~~~~~~~~~
+
+Find information about the container you want to move.
+By running ``openio container locate <container>`` you will get the list of all meta2 service address and base name.
+You can get list of container using ``openio container list``.
```
Comment (jkasarherou) [inline doc/source/admin-guide/operations_move_meta2.rst]: This is only valid for a single account...

File: doc/source/admin-guide/operations_move_meta2.rst [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+The base name is composed by CID and SEQ. The CID is before "." and SEQ is after.
+
+If the base name is 43839DACDD060FA939FAE2714A60640BDC5AFFDDBE8C494BDAB7AA994C3190A5.1,
+
+the CID is 43839DACDD060FA939FAE2714A60640BDC5AFFDDBE8C494BDAB7AA994C3190A5 and the SEQ is 1.
+
+You can launch mover by running
+  .. code-block:: console
+
+    # oio-meta2-mover <namespace> <CID> <IP source>
```
Comment (jkasarherou) [inline doc/source/admin-guide/operations_move_meta2.rst]: Again `IP source` is not a IP ...

File: doc/source/admin-guide/operations_move_meta2.rst [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+~~~~~~~~~~~
+
+Find information about the container you want to move.
+By running ``openio container locate <container>`` you will get the list of all meta2 service address and base name.
+You can get list of container using ``openio container list``.
+
+Launch mover
+~~~~~~~~~~~~
+
+The base name is composed by CID and SEQ. The CID is before "." and SEQ is after.
```
Comment (jkasarherou) [inline doc/source/admin-guide/operations_move_meta2.rst]: Might be easier to just say: format name is `{cid}.{seq}`

Comment (jkasarherou) [reply doc/source/admin-guide/operations_move_meta2.rst]: Explain what is the `SEQ`

File: doc/source/admin-guide/operations_move_meta2.rst [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+
+    # oio-meta2-mover <namespace> <CID> <IP source>
+
+This command move all sequences, but you can add the sequence number to only move one sequence.
+The destination service is automatically selected.
+
+Move to specific destination
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+You can specifify a destination where you want to move your container.
+Add the IP of a service destination service on parameter, if destination isn't free or up, an error will occured.
```
Comment (jkasarherou) [inline doc/source/admin-guide/operations_move_meta2.rst]: `occured` -> `occur`
Show one output example of such case.

**Pull request merged**